src/core/Akka/Configuration/Pigeon.conf

```diff
@@ -377,6 +386,7 @@ akka {
         parallelism-factor = 1.0
         parallelism-max = 64
       }
+      channel-executor.priority = "high"
```

Comment:
Is this meant to be `channel-scheduler` or `channel-executor`? The section above where all of the defaults are defined is `channel-scheduler`

Reply:
This are to separate components.

The `channel-scheduler` is a singleton extension (can be even standalone)
It hosts all priority task schedulers and runs the workers

The `channel-executor` is a normal akka executor component like the `ForkJoinExecutor`
It has currently only one config field: the priority. 
With the priority field it can pick the TaskScheduler from the ChannelScheduler